test/unit/DrbbResult.test.js

```diff
@@ -20,7 +20,7 @@ describe('DrbbResult', () => {
     });
 
     it('should have a link with .drbb-result-title class', () => {
-      expect(component.findWhere(n => n.text() === 'The Blithedale romance, by Nathaniel Hawthorne.')).to.have.length(1);
+      expect(component.find('a'))
```

Comment:
Can this test fail?

Reply:
That was an accidental commit. I just learned that if you use `shallow` instead of `mount` and you want to inspect a `Link`, you have to render it or use `mount`, which makes sense. Went with the former approach and wrote a better test